java/activator/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             activator
```

Comment:
In the port this is often referred to as `typesafe-activator`. Which name seems to suit better? (I don't know the software at all and don't know how people call it and what typesafe or what activator stands for.)